metrics/config.go

```diff
@@ -57,9 +57,15 @@ const (
 	Stackdriver metricsBackend = "stackdriver"
 	// Prometheus is used for Prometheus backend
 	Prometheus metricsBackend = "prometheus"
+	// OpenCensus is used to export to the OpenCensus Agent / Collector,
+	// which can send to many other services.
+	OpenCensus metricsBackend = "opencensus"
 
 	defaultBackendEnvName = "DEFAULT_METRICS_BACKEND"
 
+	collectorAddressKey = "metrics.collector"
```

Comment:
I have found that exporting config-observability field names is useful once crossing repo boundaries. In particular for writing tests in knative/serving & passing queue-proxy EnvVars

Reply:
Done.